source/raymathext.d

```diff
@@ -75,6 +75,13 @@ mixin template Linear()
             enum fragment = [FieldNameTuple!T].map!(field => field ~ op ~ "rhs." ~ field).join(",");
             return mixin("T(" ~ fragment ~ ")");
         }
+
+        T opOpAssign(string op)(inout T rhs) if (["+", "-", "*", "/"].canFind(op))
```

Comment:
These should not include multiplication or division. The omission of those in the opBinary function is on purpose.

Reply:
Also, this should be const, not inout.

Reply:
mutiplication may be used for dot product

I think the `opBinary` used inout to get compile-time evaluation (requires immutable) going.

Reply:
There is already a `dot` function.

Don't worry about `inout`, I have to fix them all anyway. And `const` covers all flavors of mutability, even for CTFE.